pages/01.guides/03.core/05.ContinuationTokens/doc.md

```diff
@@ -0,0 +1,47 @@
+---
+title: 'Continuation Tokens'
+---
+
+# Continuation Tokens
+
+Continuation tokens are a method used for Pagination in endpoints starting with V2.
+
+When you make a request to an endpoint that supports continuation tokens you will get the response and a continuation token back. This is usually found in the headers and often included in a link header.
+
+## Example
+
+```HTTP
+GET /api/v2/people HTTP/1.1
+Accept: application/json, */*
+Host: mixer.com
+link: </api/v2/chat/160788/users?limit=50&showLurkers=False&continuationToken=ABC123>; rel = "next"
```

Comment:
no space between the `=` sign https://tools.ietf.org/html/rfc5988

Reply:
good catch